plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -7806,6 +7811,14 @@ private List<Long> getTableHistory(String tableName)
                 .collect(toImmutableList());
     }
 
+    private List<Long> getTableMetadataLogEntries(String tableName)
```

Comment:
The method name is slightly misleading.
The method retrieves the `latest_sequence_number` from the metadata log entries, and not the entire entries.

Reply:
That's the same as `getTableHistory`. The function is simple enough to read through in a few seconds. And I don't think we want a super long function name. Change it to `getLatestSequenceNumber` is a bit lack of context.
https://github.com/trinodb/trino/blob/1503ecc1182fca9f38780fe5b2e3715d6766ccaf/plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java#L7806-L7808

Reply:
+1 to rename the method, maybe `getLatestSequenceNumbersInMetadataLogEntries`.